modules/onboarding-simulation/infrastructure/mongodb/autonomous-simulation-job-repository-mongodb.ts

```
@@ -0,0 +1,89 @@
+import { type Collection, type Db } from "mongodb";
```

Biome (2.4.4)

[error] All these imports are only used as types.

(lint/style/useImportType)